src/sql/base/browser/ui/infoBox/media/infoBox.css

```diff
@@ -38,7 +37,17 @@
 }
 
 .infobox-text {
-	flex: 1 1 auto;
-	padding: 10px 0;
+	flex: 0 0 auto;
+	padding: 10px 0px 0px 0px;
 	user-select: text;
+	overflow: scroll;
+	max-width: calc(100% - 75px);
+}
+
+.infobox-clickable-arrow {
+	padding: 10px 0px 0px 0px;
+}
+
+.infobox-container:hover .infobox-clickable-arrow {
+	border: 1px dashed;
```

Comment:
It seems kind of odd to have the arrow be highlighted separately but you can't actually tab onto it or interact with it directly. Is the portal doing this or something? Not sure it's necessary to have this border at all IMO.